css/index.css

```diff
@@ -143,4 +149,155 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+.nav-main {
+    display: flex;
+    align-items: baseline;
+    width: 100%;
+    height: 110px;
+    margin-top: 5px;
+}
+
+.nav-main a {
+    margin: 40px;
+    font-weight: bold;
+    color: black;
+    font-size: 18px;
+    text-decoration-line: none;
+}
+
+.jumbo-img {
+    width: 100%;
+    height: 160px;
+    margin-bottom: 20px;
+}
+
+.about-section {
+    font-weight: bold;
+    margin-bottom: 20px;
+    padding: 20px;
+    border-bottom: dotted black 2px;
+}
+
+.card-section {
+    display: flex;
+    align-content: space-evenly;
+    flex-flow: wrap;
+}
+
+ .card {
+     width: 45%;
+     font-weight: bold;
+     border-bottom: dotted black 2px;
+     padding-bottom: 20px;
+     margin-bottom: 50px;
+     margin: 15px;
+ }
+
+ .card-section .card img {
+     width: 100%;
+     height: auto;
+ }
+
+ .card h2 {
+     margin-top: 10px;
+     font-size: 20px;
+ }
+
+ .card button {
+     background: white;
+     border-radius: 10px;
+     border-color: black;
+     color: blue;
+     height: 40px;
+     width: 130px;
+     margin-top: 20px;
+     font-weight: bold;
+ }
+
+ .about-bottom-section {
+     padding: 20px;
+     margin-top: 20px;
+     margin-bottom: 20px;
+     font-weight: bold;
+ }
+
+ .container .products-about-section {
+     margin-bottom: 50px;
+ }
+
+ .container .products-about-section h1{
+     text-align: center;
+     font-weight: bold;
+     font-size: 40px;
+     text-decoration-line: underline;
+ }
+
+ .products-card-section {
+    display: flex;
+    align-content: space-evenly;
+    flex-flow: wrap;
+ }
+
+ .products-card-section .card {
+    width: 45%;
+    font-weight: bold;
+    border-bottom: dotted black 2px;
+    padding-bottom: 20px;
+    margin-bottom: 50px;
+    margin: 15px;
+    text-align: center;
+ }
+
+ .products-card-section button {
+    background: blue;
+    color: white;
+    border-radius: 10px;
+    border-color: black;
+    height: 40px;
+    width: 130px;
+    margin-top: 20px;
+    font-weight: bold;
+ }
+
+ .products-card-section button:hover {
+     background: red;
+     color: white;
+ }
+
+ .products-card-section img {
+     width: 200px;
+     height: auto;
+ }
+
+ .explosion {
+    transform: translate(258px);
+    width: 5px;
+    height: 5px;
+    border-radius: 50%;
+    transition: 1s;
+  }
+  
+  .explosion:hover {
+    box-shadow: 0 0 0 30px rgba(236, 8, 8,.1), 
+                0 0 0 60px rgba(236, 8, 8,.1), 
+                0 0 0 120px rgba(236, 8, 8,.1),
+                0 0 0 240px rgba(236, 8, 8,.1),
+                0 0 0 480px rgba(236, 8, 8,.1),
+                0 0 0 960px rgba(236, 8, 8,.1);
+  }
+
+  @keyframes nav-anim {
+    from {background-color: lightblue;}
+    to {background-color: black;}
```

Comment:
consider also having text change color if you're going from a lightblue to a black button. also, consider not having buttons change colors infinitely, but in response to user interactions or to let them know about the site's functionality (I know you were stretching on this one, just thought I'd give a couple thoughts!)